great_expectations/rule_based_profiler/parameter_builder/numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         n_resamples: Optional[Union[str, int]] = None,
         random_seed: Optional[Union[str, int]] = None,
         quantile_statistic_interpolation_method: str = "auto",
-        quantile_bias_correction: bool = False,
+        quantile_bias_correction: Union[str, bool] = False,
```

Comment:
What is this for? Are we allowing "True" or "False" to be passed as a string through kwargs?

Reply:
@NathanFarmer We always allow `$variable.*` syntax...